README.md

```diff
@@ -430,6 +430,17 @@ But if you prefer to use environment variables:
 
     export CONAN_PIP_INSTALL="bincrafters-package-tools==0.17.0,conan-promote=0.1.2"
 
+### Passing additional Docker parameters during build
+When running `conan create` step in Docker, you might want to run the container with additional parameters: for example, with `.conan/data` directory being mounted to parent system to avoid redownloading/rebuilding dependencies. For this you can use `docker_build_options` parameter
+
+    builder = ConanMultiPackager(
+      docker_build_options='--mount type=bind,source=$HOME/cached/conan-data,destination=/home/conan/.conan/data',
```

Comment:
I think the correct name should be `docker_run_options`, since `docker build` is a valid command, but we don't use it here.

Reply:
both names seem a bit tricky to me since there are multiple steps performing `docker run` in CPT